src/snowflake/cli/_plugins/nativeapp/codegen/compiler.py

```diff
@@ -34,7 +36,7 @@
 )
 
 SNOWPARK_PROCESSOR = "snowpark"
-NA_SETUP_PROCESSOR = "native-app-setup"
+NA_SETUP_PROCESSOR = "native app setup"
```

Comment:
Somehow I keep forgetting that natural strings like this are more natural in YAML.